components/Drop/MintForm/Ended.tsx

```diff
@@ -0,0 +1,77 @@
+import {
+  Box,
+  Button,
+  Heading,
+  HStack,
+  SimpleGrid,
+  Text,
+} from '@chakra-ui/react'
+import BigNumber from 'bignumber.js'
```

Comment:
Please use BigNumber from ethers whenever you don't need to deal with decimals:
```suggestion
import { BigNumber } from '@ethersproject/bignumber'
```

Reply:
[Remove bignumber.js imports](https://github.com/liteflow-labs/starter-kit/pull/359/commits/80d377f017d0d7ebae6617c8c386a018de3c7e6a)